packages/repository-json-schema/src/keys.ts

```diff
@@ -15,6 +19,6 @@ export const enum MODEL_TYPE_KEYS {
  * Metadata key used to set or retrieve repository JSON Schema
  */
 export const JSON_SCHEMA_KEY = MetadataAccessor.create<
-  {[options in MODEL_TYPE_KEYS]: JSONSchema},
```

Comment:
Any specific reason for removing this type check ?

Reply:
In the future, we will be adding more schema options like `exclude` (see https://github.com/strongloop/loopback-next/issues/2653). Because options can be combined, e.g. `partial` with `exclude`, the number of possible combinations (and thus cache) keys is growing fast. I find it impractical to enumerate all valid keys in a TypeScript enum type. We will need 4 entries now, 8 entries when we implement `exclude`, 16 entries when we introduce another option, etc.

That's why I switched the indexer type from `options in MODLE_TYPE_KEYS` to `key: string`.

Reply:
Makes sense.